### DISCUSSION TITLE
[meta] When, if ever, should I close an answered question as resolved?

### DISCUSSION THREAD

**Discussion by EliahKagan** [GitoxideLabs/gitoxide#1527] [Q&A] [answered]
GitHub supports closing a discussion as resolved. This removes it from the front page, but it is still fortunately possible to access it (including without being logged in) and to include it in searches.

When I ask a question by opening a Q&A discussion and it is answered, should I close the discussion?

It seems to me that many and probably most answered questions should never be closed, but that some can be closed if they are of less interest (e.g. #1512, and maybe this question itself once answered), if they ended up functioning like issues that were completed (e.g. #1497, #1514), or if they turn out not to be about gitoxide (e.g. #1392).

To be clear, I am not requesting that any questions be closed. Instead, I wanted to check before myself closing some (but far from all) questions I have asked, received answers to, and consider resolved.

**Comment by Byron** [comment, marked as answer]
Thanks for asking my first meta-question :)!

> It seems to me that many and probably most answered questions should never be closed, but that some can be closed if they are of less interest (e.g. #1512, and maybe this question itself once answered), if they ended up functioning like issues that were completed (e.g. #1497, #1514), or if they turn out not to be about gitoxide (e.g. #1392).

I like the idea of seeing some questions more of an issue, that then likewise should be closed to be less prominent - it seems like a practical rule to go by as well.

Personally I never asked that question and never closed such ~~issues~~ questions, but I also never tried to curate discussions in any way. Closing will definitely help in general to make *inactive* discussions more obvious, which might be another way to treat them.

**Reply by EliahKagan** [reply]
Sounds good. I'll close some of my own answered questions along those lines, but I'll be conservative in doing so.
